wiki/Rules/en.md

```diff
@@ -5,7 +5,7 @@ These rules are the basis from which we help keep the osu! community a fun and w
 ## Community rules
 
 1. **Each player may only have *ONE* account, ever.** The *first* account you make during account registration is your one and only osu! account, from the moment of its creation and forevermore. This account is YOU. It is not anyone else — not your brother, your mother, your sister, your friend — it is YOU. Don't share your account with anyone else. You don't get to make a new one if you lose access to it. Keep it safe.
-2. **Play fair.** Using third-party utilities of any kind to get any sort of advantage is not okay. This includes things like macro programs, aim-assist programs, timescale modification, and so on. If a program is doing something to help you play the game that you should be doing yourself, it isn't okay!
+2. **Play fair.** Using third-party utilities of any kind to get any sort of advantage is not okay. This includes things like macro programs, aim-assist programs, timescale modification, and so on. Macros, keybinds, or hardware where a physical action (e.g. pressing a key down) does not correspond to the same in-game action are similarly disallowed.[^macros] If a program is doing something to help you play the game that you should be doing yourself, it isn't okay!
```

Comment:
```suggestion
2. **Play fair.** Using third-party utilities of any kind to get any sort of advantage is not okay. This includes things like aim-assist programs, timescale modifications, and so on. Macros, keybinds, or hardware where a physical action (e.g. pressing a key down) does not correspond to the same in-game action are similarly disallowed.[^macros] If a program is doing something to help you play the game that you should be doing yourself, it isn't okay!
```
I think it's now unnecessary to say macro programs there because you say it again next sentence in a way that sounds like you're adding to the list, not reiterating

Reply:
yup good point